.vscode/settings.json

```diff
@@ -1,5 +1,6 @@
 {
     "editor.codeActionsOnSave": {
         "source.fixAll.eslint": true
-    }
+    },
+    "files.insertFinalNewline": true,
 }
```

Comment:
The irony is strong

Reply:
Is this not what you meant?

Reply:
Heh yesbut:
<img width="494" alt="image" src="https://user-images.githubusercontent.com/730704/170400722-5c21c57f-4b9c-430a-9054-2d540708b92c.png">